types/node/process.d.ts

```diff
@@ -274,10 +275,8 @@ declare module "process" {
                 | "loong64"
                 | "mips"
                 | "mipsel"
-                | "ppc"
```

Comment:
Are things typically _removed_ from this list? I can imagine this being a breaking change for some people switching over this.

Reply:
There isn't a precedent since this union was introduced. [vscode-zig](https://github.com/ziglang/vscode-zig) is the only significant repo with affected usage that I could find, although they have their dependency pinned to v20. I get what you mean in terms of libraries not being able to test for those values if they aim to support older versions, although such a build probably shouldn't be typechecking against the latest API definitions anyway.

It's not a hill I'd die on!

Reply:
> I get what you mean in terms of libraries not being able to test for those values if they aim to support older versions, although such a build probably shouldn't be typechecking against the latest API definitions anyway.

TS uses `@types/node` at latest, yet works in Node 14 😄

Notably this lib will be picked up by ATA, so, it's sort of a thing there. I don't know how much it really matters.

Reply:
Looks like the documentation for `os.arch()` and `process.arch` became out-of-sync. V8 removed build support, though, so it's definitely gone.

Easiest to revert until v22 falls EOL?

Reply:
Eh, I think it's probably fine to leave as-is and then see what happens.